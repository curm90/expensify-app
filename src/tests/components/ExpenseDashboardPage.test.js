import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from '../setupTests';
import Homepage from '../../components/ExpenseDashboardPage';

it('should render dashboard page', () => {
  const wrapper = shallow(<Homepage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
