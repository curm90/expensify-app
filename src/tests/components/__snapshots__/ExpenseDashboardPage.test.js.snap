// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard page 1`] = `
<div>
  <h1>
    Homepage
  </h1>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
